src/builder/command.rs

```diff
@@ -1612,6 +1622,33 @@ impl Command {
     ///     .override_usage("myapp [-clDas] <some_file>")
     /// # ;
     /// ```
+    ///
+    /// Or for multiple usage lines:
+    ///
+    /// ```no_run
+    /// # use clap::{Command, Arg};
+    /// Command::new("myprog").override_usage(
+    ///     "\
+    ///     myapp -X [-a] [-b] <file>
+    ///     myapp -Y [-c] <file1> <file2>
+    ///     myapp -Z [-d|-e]"
+    /// )
+    /// # ;
+    /// ```
+    ///
+    /// or equivalently, but with different string escaping:
+    ///
+    /// ```no_run
+    /// # use clap::{Command, Arg};
+    /// Command::new("myprog")
+    ///     .override_usage(
+    ///         "myapp -X [-a] [-b] <file>\n    \
+    ///          myapp -Y [-c] <file1> <file2>\n    \
+    ///          myapp -Z [-d|-e]"
+    ///     )
+    /// # ;
+    /// ```
```

Comment:
By total volume of documentation, I feel like having two code samples for this is probably too much

Reply:
Fair. There is now one code example for multiple lines. I chose the one with uglier, but less ambiguous string escaping.